### DISCUSSION TITLE
Access to medianav

### DISCUSSION THREAD

**Discussion by alex-military** [cedricp/ecutweaker#21] [Q&A]
Hi, I tried to access channel 2 via a modified cable to connect to my medianav but I can't do anything. I wanted to know if the Android version can connect to channel 2 and if I need to do something first or if I have to try with ddt4all.

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/970

**Comment by alex-military** [comment]
hi thanks for the reply but this doesn't answer my question. what i need to know is if it works with the ecutweaker app or if i have to use ddt4all.
